wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/watcher/server/impl/FileWatcherService.java

```diff
@@ -269,33 +270,35 @@ private void run() {
           continue;
         }
 
-        List<WatchEvent<?>> watchEvents = watchKey.pollEvents();
+        for (List<WatchEvent<?>> watchEvents = watchKey.pollEvents();
```

Comment:
maybe `while` statement here make code more readable ?

Reply:
You mean something like this?:

```
        List<WatchEvent<?>> watchEvents = watchKey.pollEvents();
        while(!watchEvents.isEmpty()) {
          if (suspended.get()) {
            ...
          }

          for (WatchEvent<?> event : watchEvents) {
            ...
          }

          watchEvents = watchKey.pollEvents();
        }
```